### ISSUE TITLE
gzip.open(filename, "rt") fails on Python 2.7 on Windows

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#1138]
This is a Python bug, logged as http://bugs.python.org/issue30012

Right now it shows up as a failure on our test suite,

```
======================================================================
ERROR: test_doctests (test_Tutorial.TutorialTestCase)
Run tutorial doctests.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\repositories\BuildBotBiopython\win27\build\Tests\test_Tutorial.py", line 228, in test_doctests
    (len(failures), ", ".join(failures)))
ValueError: 1 Tutorial doctests failed: test_chapter_seqio_line_00326

----------------------------------
```

Example from http://testing.open-bio.org/biopython/builders/Windows%20XP%20-%20Python%202.7/builds/796

**Comment by peterjc** [comment]
The Python decided this was a feature request, and that the difference in behaviour on Windows Python 2.7 is not actually a bug. This will not be fixed in Python 2.7 given its maintenance status.

The simplest 'fix' is to stop using this tutorial example as a doctest since it will fail on our BuildBot etc.

We could further enhance ``test_Tutorial.py`` to offer Python version conditional checks, which might be helpful with a few other Python 2 vs 3 cases?

**Comment by peterjc** [comment]
I've disabled the problematic test for now, and am closing this.
